tests/unit/Node/StringLiteralTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
```

Comment:
Would it make sense to use the pattern in `tests/cases/parser` for this?

Reply:
hm, in what sense? 

keeping this test case but globbing over `../cases/parser/stringLitreal.php`  or something else?

unless I'm missing something there is no way to assert anything with the parser tests as they check the AST structure, while this test is testing a method on the Node.

Reply:
Sorry, I didn't look at it closely enough, yes this makes sense as-is.